### DISCUSSION TITLE
Intrusive errors when setting custom apex domain for github pages

### DISCUSSION THREAD

**Discussion by mkungla** [community/community#5644] [Other Feature Feedback, Questions, & Ideas]
When adding custom domains to GitHub pages you get immediately 2 Intrusive errors e.g.

----

error

**Both howijd.org and its alternate name are improperly configured**  
Domain does not resolve to the GitHub Pages server. For more information, see Learn more  
(NotServedByPagesError). We recommend you add an A record pointed to our IP addresses, or an ALIAS record  
pointing to howijd.github.io.

and error on top

**Domain howijd.org is not eligible for HTTPS at this time.**

---

While in fact everything is configured correctly

```
dig howijd.org +noall +answer -t A
howijd.org.		10800	IN	A	185.199.110.153
howijd.org.		10800	IN	A	185.199.111.153
howijd.org.		10800	IN	A	185.199.108.153
howijd.org.		10800	IN	A	185.199.109.153
```

```
dig howijd.org +noall +answer -t AAAA
howijd.org.		10800	IN	AAAA	2606:50c0:8000::153
howijd.org.		10800	IN	AAAA	2606:50c0:8003::153
howijd.org.		10800	IN	AAAA	2606:50c0:8002::153
howijd.org.		10800	IN	AAAA	2606:50c0:8001::153
```

```
dig howijd.org +noall +answer -t CAA
howijd.org.		10800	IN	CAA	0 issue "letsencrypt.org"
```

One should just wait some minutes since it will take some time for GitHub servers to pick it up, generate certificates and deploy it. So perhaps make it a bit more clear that some patience is needed. Instead of showing scary red errors and guiding user to seek configuration issues which don't exists.

**Comment by jfullstackdev** [comment]
I think it's being seen on the Pages settings, indicating there the status of that

**Comment by trusktr** [comment]
I was having this red error in the UI, `Domain docs.lume.io is not eligible for HTTPS at this time`, making me think something was wrong. I see similar `dig` output as @mkungla.

After 15 minutes I came back, but I still saw the following in the UI upon checking the Pages page in settings:

<img width="795" alt="Screenshot 2023-01-08 at 1 27 03 AM" src="https://user-images.githubusercontent.com/297678/211189024-ed3beb1c-1b6b-4f2d-8e82-c67c47743c42.png">

When I clicked "check again" I saw this at the top:

<img width="427" alt="Screenshot 2023-01-08 at 1 28 28 AM" src="https://user-images.githubusercontent.com/297678/211189046-572253e1-28ec-4dc7-aa79-9f0b28f411ce.png">

Meanwhile the yellow warning was still showing in the UI.

A few minutes later I refreshed the Pages page and the yellow message went away, and status went green.

<img width="435" alt="Screenshot 2023-01-08 at 1 43 27 AM" src="https://user-images.githubusercontent.com/297678/211189523-c110f76d-dc5d-40b5-9316-74addf8594f8.png">

The `Your site is live at https://...` text still showed http and not https until a page refresh. I then checked "enforce https".

5 minutes later the site still showed as insecure in Chrome, and http did not redirect to https.

After 5 more minutes or so http redirected to https, but strangely Chrome showed the certificate as valid, but still insecure, with no option to re-show the warning as with invalid certs:

<img width="383" alt="Screenshot 2023-01-08 at 1 34 12 AM" src="https://user-images.githubusercontent.com/297678/211189291-b1b3fb34-9582-4aa8-9c19-f307993a2889.png">

10 minutes later, I am stuck in the valid but insecure state. https://docs.lume.io

Will check back tomorrow.

**Reply by trusktr** [reply]
Huh, still "valid" but insecure this morning.

**Reply by trusktr** [reply]
Ah, restarting Chrome was the last thing. All secure in the address bar now! That's was an odd last issue that usually doesn't happen.
